File: pytools/data/data.code.cpp.server/AckSceneExit.cpp
#ifndef _ACK_SCENE_EXIT_
#define _ACK_SCENE_EXIT_

#include <list>

#include <pb_type.hpp>
#include <Packet.hpp>

#include <Msg.hpp>



class AckSceneExit
{
private:
	U32 m_uid;


public:
	Packet Encode()
	{
		Packet packet;
		packet.WriteU32(m_uid);
		packet.Encode(Msg::P_ACK_SCENE_EXIT);
		return packet;
	}


	void SetUid(U32 uid)
	{
		m_uid = uid;
	}

};

#endif

File: pytools/data/data.code.csharp.server/AckSceneExit.cs
using System.Collections;
using System.Collections.Generic;


public class AckSceneExit
{
	private uint _uid;


	public Packet Encode()
	{
		Packet packet = new Packet();
		packet.WriteUint(this._uid);
		packet.Encode(Msg.P_ACK_SCENE_EXIT);
		return packet;
	}


	public uint uid
	{
		get { return this._uid; }
		set { this._uid = value; }
	}

}
